contracts/src/dice.cpp

```diff
@@ -2,73 +2,76 @@
 
 namespace dice {
 
-void dice::check_params(uint64_t ses_id) {
-    eosio::check(get_param_value(ses_id, min_bet_param_type) != std::nullopt, "absent min bet param");
-    eosio::check(get_param_value(ses_id, max_bet_param_type) != std::nullopt, "absent max bet param");
-    eosio::check(get_param_value(ses_id, max_payout_param_type) != std::nullopt, "absent max payout param");
+void dice::check_params(uint64_t ses_id) const {
```

Comment:
also need to add params relations checks (max_bet > min_bet, max_payout > max_bet, etc..)